force-app/main/default/classes/DMLOperations.cls

```diff
@@ -31,13 +31,14 @@ public with sharing class DMLOperations {
      */
     public static Id insertNewAccount() {
         // Create a new Account instance
-
+        Account newAccount = new Account();
+        
         // Set some fields on the Account
-
+        newAccount.Name = 'Account_' + String.valueOf(Math.random());
         // Insert the Account into the database
-
+        insert newAccount;
```

Comment:
Try to see if they user has the permission to insert an account https://salesforce.stackexchange.com/questions/92209/how-to-check-access-of-user-for-particular-user-on-object